readme.md

```diff
@@ -179,7 +179,7 @@ and detailed information on these topics:
 - [Proxy Packages](https://godoc.org/github.com/zalando/skipper/proxy)
 - [Logging](https://godoc.org/github.com/zalando/skipper/logging) and [Metrics](https://godoc.org/github.com/zalando/skipper/metrics)
 - Performance Considerations
-- [Rate Limiters](https://godoc.org/github.com/zalando/skipper/reference/filters/#ratelimit)
+- [Rate Limiters](https://godoc.org/github.com/zalando/skipper/filters/ratelimit)
```

Comment:
I think the former link should have been https://opensource.zalando.com/skipper/reference/filters/#ratelimit

Reply:
Could be. I was generally confused by two different docs being linked throughout this bullet point list seeing no clear pattern to it.

Reply:
The pattern should be:
- developer focused docs that are used by skipper as library people -> godoc, or even better the new docs page by go that moved from godoc to pkg.go.dev: https://pkg.go.dev/github.com/zalando/skipper
- user focused docs that are used by skipper as binary/container/SaaS -> https://opensource.zalando.com/skipper